Makefile.PL

```diff
@@ -49,11 +49,14 @@ my %requires = (
     'Email::Send'              => '1.911',
     'File::Slurp'              => '9999.13',
     'JSON::XS'                 => '2.01',
+    'LWP::Protocol::connect'   => 0,
```

Comment:
This is the only line of this patch I'm truly comfortable saying "Yes, this looks correct, and I understand what will happen here."